<?xml version="1.0"?>
<launch>

    <arg name="serial_port"/>

    <node pkg="cm_bridge" name="cm_api_manager" type="cm_api_manager.py">
        <param name="serial_port" value="$(arg serial_port)"/>
    </node>

</launch>
